Fix-88 Starts building image-inspector image at registry.centos.org

navidshaikh opened the pull request:
Fixes #88

  Adds a file cccp.yml with `job-id: image-inspect`, needed to build
  at registry.centos.org

enoodle commented:
@navidshaikh Thanks,

Can you later add this to https://github.com/CentOS/container-index ?

simon3z commented:
@enoodle can you review?

navidshaikh commented:
> Can you later add this to https://github.com/CentOS/container-index ?

Raised a PR  https://github.com/CentOS/container-index/pull/250 , what should be the preferable email to receive build notifications updates on?

enoodle commented on Dockerfile:
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM registry.centos.org/centos/centos:latest
```
Why use `latest` instead of `7` tag?

enoodle commented:
> what should be the preferable email to receive build notifications updates on?

You can put @simon3z 's if he wants it there or otherwise mine.

navidshaikh commented:
> Why use latest instead of 7 tag?

Updated the tag to `7`, base image is now `registry.centos.org/centos/centos:7`

> You can put @simon3z 's if he wants it there or otherwise mine.

Updated https://github.com/CentOS/container-index/pull/250 with @enoodle email for now, I will update as @simon3z confirms.
